appeals/web/src/server/appeals/appeal-documents/appeal-documents.controller.js

```diff
@@ -44,9 +44,12 @@ import { permissionNames } from '#environment/permissions.js';
  * @param {string} [params.nextPageUrl]
  * @param {boolean} [params.isLateEntry]
  * @param {string} [params.pageHeadingTextOverride]
+ * @param {string?} [params.uploadContainerHeadingTextOverride]
+ * @param {string?} [params.documentTitle]
```

Comment:
I don't think `{string?}` is a valid type here. Putting the arg name in square brackets as you have marks it as optional.

```suggestion
 * @param {string} [params.uploadContainerHeadingTextOverride]
 * @param {string} [params.documentTitle]
```

Reply:
Changed as suggested